packages/components/button/stories/IconButton.stories.tsx

```diff
@@ -46,6 +46,38 @@ basic.args = {
   variant: 'transparent',
 };
 
+export const withTooltip = ({ icon, iconProps, ...props }) => (
```

Comment:
Can we have this story—or an additional one—default to tooltip open so we can pick it up with Chromatic?